j2k/newSrc/org/jetbrains/kotlin/j2k/JKSymbolProvider.kt

```diff
@@ -73,6 +77,14 @@ class JKSymbolProvider {
         JKUniverseMethodSymbol().also { it.target = jk }
     } as JKMethodSymbol
 
+    internal inline fun <reified T : JKSymbol> provideByFqName(fqName: String): T = symbolByFqName.getOrPut(fqName) {
+        resolveFqName(ClassId.fromString(fqName), symbolsByPsi.keys.first())?.let(this::provideDirectSymbol) as? T ?: when {
```

Comment:
WTF happened with context? Why it is just first of Psi keys, this can be unstable